### DISCUSSION TITLE
OAuth authorization vendor users with a public App

### DISCUSSION THREAD

**Discussion by yigal-mish** [amzn/selling-partner-api-models#3321] [General] [answered]
Hi,

I created a public App that uses Selling Partner API. 
Our intention is that our App will work with Sellers and Vendors.
It authenticates through Seller Central with the following URL: https://sellercentral.amazon.com/apps/authorize/consent 
This URL doesn't work for Vendors.
Is there any way to authenticate **Vendor** users with a public App?

**Comment by vaidas-lungis** [comment, marked as answer]
<img width="843" alt="Screenshot 2022-05-10 at 08 57 00" src="https://user-images.githubusercontent.com/5111555/167558195-ba77bd35-5f7e-442e-b1a9-65ae99a94f1f.png">

https://developer-docs.amazon.com/amazon-shipping/docs/website-authorization-workflow#step-1-the-selling-partner-initiates-authorization-from-your-website

https://developer-docs.amazon.com/amazon-shipping/docs/vendor-central-urls

But Vendor Central URL gives 404 😭
